main.go

```diff
@@ -210,6 +210,8 @@ func setupControllers(ctx context.Context, mgr ctrl.Manager, cCache *cache.Cache
 	}
 
 	crds := findCustomResources(ctx, mgr)
+	// Pods are always here
+	crds.Insert("pod")
```

Comment:
I think for Job we did it inside `findCustomResources`. Maybe we can change the function to `availableJobResourceDefinitions`, as it's not just about Custom Resources.

Looking at the code, you probably thought about this already, but just to make sure:
We should not enable "pod" support by default, as we did for all other CRDs.